### PR TITLE
Fixes the post_elements return

### PR DESCRIPTION
**Pull request by davidmokos**
We can now use the selenium driver.find_elements_by_id() without problems.

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
@davidmokos Please sign the CLA, so we can merge your PR.

**Comment by davidmokos** [comment]
@mykola-mokhnach I have already signed the CLA. If I open it, it is signed and I can't do any other action. Any advise?

**Comment by mykola-mokhnach** [comment]
Make sure you have the correct email address set in your local git config (the same that was used to sign the CLA)

**Comment by davidmokos** [comment]
I added the email to my account. Is everything correct now?

**Comment by mykola-mokhnach** [comment]
try to push a new commit with the updated email address or recreate the PR

**Comment by davidmokos** [comment]
Now it seems to be ok.

**Comment by davidmokos** [comment]
Can you merge the changes and close the pull request?

**Comment by davidmokos** [comment]
Can you merge the changes and close the pull request?

File: AppiumForMac/Server/Handlers/AfMHandlers.m [hunk 1]
```diff
@@ -543,6 +543,9 @@ - (AppiumMacHTTPJSONResponse *)post_elements:(NSString*)path data:(NSData*)postD
                     [elements addObject:[NSDictionary dictionaryWithObject:myKey forKey:@"ELEMENT"]];
                 }
                 
+                return [AppiumMacHTTPJSONResponse responseWithJson:elements status:kAfMStatusCodeSuccess session:session.sessionId];
+                
+            } else {
```
Comment (mykola-mokhnach) [inline AppiumForMac/Server/Handlers/AfMHandlers.m]: this `else` is, actually, not needed

Comment (davidmokos) [reply AppiumForMac/Server/Handlers/AfMHandlers.m]: I think it is is better to respond with status `kAfMStatusCodeNoSuchElement` when no element is found.

Comment (mykola-mokhnach) [reply AppiumForMac/Server/Handlers/AfMHandlers.m]: I mean you could just simplify it to 

```
...
return [AppiumMacHTTPJSONResponse responseWithJson:elements status:kAfMStatusCodeSuccess session:session.sessionId];
}
...
return [AppiumMacHTTPJSONResponse responseWithJson:nil status:kAfMStatusCodeNoSuchElement session:session.sessionId];
```

Comment (davidmokos) [reply AppiumForMac/Server/Handlers/AfMHandlers.m]: Yes, you are right

**Pull request merged**